x/tokenfactory/keeper/createdenom.go

```diff
@@ -64,3 +74,33 @@ func (k Keeper) validateCreateDenom(ctx sdk.Context, creatorAddr string, subdeno
 
 	return denom, nil
 }
+
+func (k Keeper) chargeForCreateDenom(ctx sdk.Context, creatorAddr string) (err error) {
```

Comment:
since all of that happening exists in `createdenom` file, how about renaming function to `chargeFee`?